dali/test/python/test_pipeline.py

```diff
@@ -408,6 +416,29 @@ def iter_setup(self):
             img_chw = img_chw_test
         assert(np.sum(np.abs(img_chw - img_chw_test)) == 0)
 
+def test_make_continious_serialize():
+    class COCOPipeline(Pipeline):
+        def __init__(self, batch_size, num_threads, device_id):
+            super(COCOPipeline, self).__init__(batch_size, num_threads, device_id)
+            self.input = ops.COCOReader(file_root=coco_image_folder, annotations_file=coco_annotation_file, ratio=True, ltrb=True)
+            self.decode = ops.nvJPEGDecoder(device="mixed")
+            self.crop = ops.RandomBBoxCrop(device="cpu")
+            self.slice = ops.Slice(device="gpu")
+
+        def define_graph(self):
+            inputs, bboxes, labels = self.input()
+            images = self.decode(inputs)
+            crop_begin, crop_size, bboxes, labels = self.crop(bboxes, labels)
+            images = self.slice(images, crop_begin, crop_size)
+            return images
+
+
+    pipe = COCOPipeline(batch_size=32, num_threads=2, device_id=0)
+    serialized_pipeline = pipe.serialize()
+    del(pipe)
+    new_pipe = Pipeline(batch_size=32, num_threads=2, device_id=0)
+    new_pipe.deserialize_and_build(serialized_pipeline)
```

Comment:
how about testing that both pipelines produce the same outputs for the same seed?

Reply:
Done in next test case. In that one I want to have `pipe` deleted before creating new one.